https.go

```diff
@@ -453,6 +453,11 @@ func (proxy *ProxyHttpServer) NewConnectDialToProxyWithHandler(https_proxy strin
 }
 
 func TLSConfigFromCA(ca *tls.Certificate) func(host string, ctx *ProxyCtx) (*tls.Config, error) {
+	ca.Certificate = [][]byte{ca.Certificate[0]}
+	return TLSConfigFromCANative(ca)
+}
+
+func TLSConfigFromCANative(ca *tls.Certificate) func(host string, ctx *ProxyCtx) (*tls.Config, error) {
```

Comment:
Can't we have just a single TLSConfigFromCA function that returns the whole chain? This shouldn't be a breaking change